py-utils/test/test_conf_store/test_conf_store.py

```diff
@@ -0,0 +1,22 @@
+from cortx.utils.conf_store import ConfStore, KvStore
+from cortx.utils.conf_store import KvStorageFactory
+
+
+def example_function(path):
+    # local / global
+    # conf = ConfStore('local')
+    conf = ConfStore()
+    kv_store_backend = KvStorageFactory(path)

```

Comment:
We can keep KvStorage transparent for users. It can be like this... 

```
conf = ConfStore()

def load_config(index, backend_url): 
    # Instantiate and Load Config  
    conf_backend = KvStoreFactory(backend_url)
    conf.load(index, kvstore=conf_backend)

# Load configs
load_config('sspl_local', 'file:///etc/cortx/sspl.json')
load_config('sspl_global', 'console://consule.xyz.com:5050/sspl')

# Use Configuration 
user_name = conf.get('sspl_local', 'management.user', default_value=None)
enclosure_ip = conf.get('sspl_global', 'enclosure.ip', default_value='127.0.0.1')
...
```